### PR TITLE
updated-documentation

### PR DESCRIPTION
**Pull request by arcoprova**
id-page-image1-before
id-page-image1-now
id-page-image2-before
id-page-image2-now
id-page-image3-before
id-page-image3-now
Removed horizontal scrollbar by giving scale to images.
input-page-before
input-page-now
updated documentation on input page.
text-page-output1-before
text-page-output1-now
text-page-output2-before
text-page-output2-now
text-page-output3-before
text-page-output3-now
text-page-output4-before
text-page-output4-now
updated text-color on text page.

### CODE REVIEW

File: id.ftd [hunk 1]
```diff
@@ -205,3 +209,5 @@ id available for element which you want to refer to.
 
 -- ft.image:
 src: $assets.files.images.ftd-give-me-an-element-with-id.png
+scale: 1
```
Comment (Arpita-Jaiswal) [inline id.ftd]: Why `scale: 1` ?

File: id.ftd [hunk 2]
```diff
@@ -89,6 +89,8 @@ these.
 
 -- ft.image:
 src: $assets.files.images.ask-id-to-ftd.png
+scale: 1
```
Comment (Arpita-Jaiswal) [inline id.ftd]: Why `scale:1`?

File: id.ftd [hunk 3]
```diff
@@ -131,6 +133,8 @@ which becomes `id` for DOM node of `ftd.text`element. 😊 )
 
 -- ft.image:
 src: $assets.files.images.ftd-give-me-id-yes-got-it.png
+scale: 1
```
Comment (Arpita-Jaiswal) [inline id.ftd]: Same as above

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: id.ftd (hunk 1 above)
Comment (arcoprova) [reply id.ftd]: after giving scale: 1, the image size issues have solved.

File: id.ftd (hunk 2 above)
Comment (arcoprova) [reply id.ftd]: after giving scale: 1, the image size issues have solved

File: id.ftd (hunk 3 above)
Comment (arcoprova) [reply id.ftd]: after giving scale: 1, the image size issues have solved

**Review by Arpita-Jaiswal — APPROVED** [review]

**Pull request merged**